classylss/binding.pyx

```diff
@@ -571,14 +589,14 @@ cdef class Background:
         Density of matter :math:`\rho_b` as a function of redshift, in
         units of :math:`10^{10} (M_\odot/h) (Mpc/h)^{-3}`.
         """
-        return self.Omega_m(z) * self.rho_tot(z)
+        return self.Omega_m(z) * self.rho_crit(z)
 
     def rho_r(self, z):
         r"""
         Density of radiation :math:`\rho_r` as a function of redshift, in
         units of :math:`10^{10} (M_\odot/h) (Mpc/h)^{-3}`.
         """
-        return self.Omega_r(z) * self.rho_tot(z)
+        return self.Omega_r(z) * self.rho_crit(z)
```

Comment:
Did you check if this is in class? I think the definition in class is against rho_tot, not rho_crit (I may have miss looked).